src/components/web3/ExceptionListPanel.test.ts

```diff
@@ -0,0 +1,40 @@
+import { splitAddresses } from "./api";
```

Comment:
if the code is in `./api`, the tests should be in `api.test.ts` not here. Just renaming this file is fine.